scripts/specializations/CpAIWorker.lua

```diff
@@ -484,16 +495,86 @@ function CpAIWorker:stopCpDriver()
 	if actionController ~= nil then
 		actionController:resetCurrentState()
 	end
-
 	self:raiseAIEvent("onAIFieldWorkerEnd", "onAIImplementEnd")
-
 end
 
 function CpAIWorker:getCpDriveStrategy()
     local spec = self.spec_cpAIWorker
     return spec.driveStrategy
 end
 
+function CpAIWorker:getCpReverseDrivingDirectionNode()
+    local spec = self.spec_cpAIWorker
+    if not spec.reverseDrivingDirectionNode and SpecializationUtil.hasSpecialization(ReverseDriving, self.specializations) then
+        spec.reverseDrivingDirectionNode =
+            CpUtil.createNewLinkedNode(self, "realReverseDrivingDirectionNode", self:getAIDirectionNode())
+        setRotation(spec.reverseDrivingDirectionNode, 0, math.pi, 0)
+    end
+    return spec.reverseDrivingDirectionNode
+end
+
+--- Fixes the ai reverse node rotation for articulated axis vehicles,
+--- if the node is pointing backwards and not forwards.
+function CpAIWorker:getAIReverserNode(superFunc)
+    local spec = self.spec_cpAIWorker
+ --   if not self:getIsCpActive() then return superFunc(self) end
+    if self.spec_articulatedAxis and self.spec_articulatedAxis.aiRevereserNode then
```

Comment:
do they use the mistyped version "revereser" everywhere? never the correct one?

Reply:
yes seems so :)